dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry10/internal/config/OpenTelemetryVersionedConfigurationImpl.java

```diff
@@ -43,4 +54,11 @@ public Map<String, String> getTelemetryPropertyDefaults() {
         telemetryProperties.put(OpenTelemetryConstants.ENV_LOGS_EXPORTER_PROPERTY, "none");
         return telemetryProperties;
     }
+
```

Comment:
Ideally we'd move this back into the info factory, but refactoring like that can be done in a follow up when we're not on a tight deadline.